java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/Application.java

```diff
@@ -57,6 +57,13 @@
         .desc("Mappings between supported jdk versions, server types, and docker images")
         .build());
 
+    CLI_OPTIONS.addOption(Option.builder("c")
```

Comment:
The README has been updated to mention it as one of the required args to the docker container. Do you think there should be additional documentation of compat?